### PR TITLE
Update `last-week-on-osm` -> `latest-changes`

### PR DESCRIPTION
**Pull request by rene78**
- Moved latest-changes to "Changeset Tools"
- Updated description
- Changed repository tyrasd --> rene78 (more up-to-date)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ We also have a list of <a href="UNMAINTAINED.md">unmaintained projects</a>. If y
 
 * [osmcha](https://osmcha.org) - Detector for suspicious changesets. ([Source Code](https://github.com/mapbox/osmcha-frontend) / [Wiki](https://wiki.openstreetmap.org/wiki/OSMCha))
 * [achavi](https://overpass-api.de/achavi/) - Augmented change viewer. ([Source Code](https://github.com/nrenner/achavi/) / [Wiki](https://wiki.openstreetmap.org/wiki/Achavi))
+* [latest-changes](https://rene78.github.io/latest-changes/#2/15.0/-15.0) - Visualize latest OSM changesets within a certain geographic boundary. Includes simple vandalism checker. ([Source Code](https://github.com/rene78/latest-changes/))
```
Comment (Unkn0wnCat) [inline README.md:115]: I agree with the recategorization, but please move this to the bottom of the category as the categories are supposed to be sorted by date added and adding in the middle (even if just for a move) would kinda undermine that 😛

**Review by Unkn0wnCat — CHANGES_REQUESTED** [review]
Looking good already. I'm on board with switching to your fork as it seems you were the only one actively contributing to the old version anyways and the original version seems to be unmaintained.

Please fix the sorting as described in the attached comment, and I'm happy to merge!

**Comment by rene78** [comment]
Thanks for the feedback! Makes sense. I have just moved it to the end of the list.

**Review by Unkn0wnCat — APPROVED** [review]

**Comment by Unkn0wnCat** [comment]
Looking good! Thanks for helping with keeping this list up to date 🥇

**Pull request merged**